api/app/signals/apps/signals/migrations/0105_auto_20200415_1603.py

```diff
@@ -0,0 +1,106 @@
+# Generated by Django 2.2.12 on 2020-04-15 14:03
```

Comment:
General remark, this migration contains a lot of other changes. So maybe split these up to have one migration that will fix al the missing stuff in other models and one migration that will have the "gebieden" stuff.

Reply:
Yes, will do.